http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueriesSpec.scala

```diff
@@ -0,0 +1,124 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package http
+package db
+
+import cats.effect.IO
+import org.broadinstitute.dsde.workbench.leonardo.CommonTestData._
+import org.broadinstitute.dsde.workbench.leonardo.config.Config
+import org.broadinstitute.dsde.workbench.leonardo.db.{
+  labelQuery,
+  LabelResourceType,
+  RuntimeServiceDbQueries,
+  TestComponent
+}
+import org.broadinstitute.dsde.workbench.leonardo.http.api.ListRuntimeResponse2
+import org.scalatest.FlatSpecLike
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class RuntimeServiceDbQueriesSpec extends FlatSpecLike with TestComponent with GcsPathUtils with ScalaFutures {
+
+  "RuntimeServiceDbQueries" should "list runtimes" in isolatedDbTest {
+    val res = for {
+      list1 <- RuntimeServiceDbQueries.listClusters(Map.empty, false, None).transaction
+      c1 <- IO(makeCluster(1).save())
+      list2 <- RuntimeServiceDbQueries.listClusters(Map.empty, false, None).transaction
+      c2 <- IO(makeCluster(2).save())
+      list3 <- RuntimeServiceDbQueries.listClusters(Map.empty, false, None).transaction
+    } yield {
+      list1 shouldEqual List.empty
+      val c1Expected = toListRuntimeResponse(c1, Map.empty)
+      val c2Expected = toListRuntimeResponse(c2, Map.empty)
+      list2 shouldEqual List(c1Expected)
+      list3.toSet shouldEqual Set(c1Expected, c2Expected)
+    }
+
+    res.unsafeRunSync()
```

Comment:
would it make sense to add a check to make sure the query doesn't take more than say 5 seconds?

Reply:
Sure could add something like that :)